package.json

```diff
@@ -11,10 +11,11 @@
     "migrations/scripts"
   ],
   "scripts": {
+    "build": "hardhat compile",
     "ganache": "echo 'GANACHE IS NO LONGER USED; use buidler-vm script instead'",
     "buidler-vm": "buidler node",
     "buidler-vm:log": "buidler node | grep '^    \\|console.log'",
-    "test": "NODE_ENV=test truffle test",
```

Comment:
Are we sure `NODE_ENV` is not used in tests anymore?

Reply:
It seems `NODE_ENV` is not needed for hardhat to run.

Reply:
Ok fine